components/org.wso2.carbon.identity.sso.saml.common/src/main/java/org/wso2/carbon/identity/sso/saml/common/Util.java

```diff
@@ -129,7 +130,7 @@ public static boolean isHttpSuccessStatusCode(int status) {
 
         return filteredProviders;
     }
-
+    @Deprecated
```

Comment:
Please add a deprecation comment on why you are deprecating this method. Also mention if this API was replaced by another API.